server/apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -175,115 +128,3 @@ def format(self, article, subscriber):
 
     def can_format(self, format_type, article):
         return format_type == 'AAP IPNEWS' and article['type'] in ['text', 'preformatted']
```

Comment:
BTW, in general it is better to use `sets` for containment testing, i.e. `if x in {'value1', 'value2'}`, because of O(1) lookup (compared to O(n) for lists/sets), but it's hardly of any significance here in this particular case.